### ISSUE TITLE
Typescript compiler fails when using react css themr with error Default export of the module has or is using private name 'TReactCSSThemrTheme'

### ISSUE THREAD

**Issue by sahilgupta202** [javivelasco/react-css-themr#73]
Typescript Version : 2.4.2
React CSS Themr: 2.1.2

**Replication Steps**

1. Create a component that uses themr.
       `themr('Identifier')(ComponentName);`
2. Run the Typescript compiler 

It will throw the following error
     ` error TS4082: Default export of the module has or is using private name 'TReactCSSThemrTheme'.`
     ` error TS4082: Default export of the module has or is using private name 'ThemedComponentClass'.
`

**Comment by belohlavek** [comment]
Quick fix: just import it ;)

**Comment by sahilgupta202** [comment]
Yes @belohlavek we used that, but that is not a good solution because TS Lint will fail as we are not using them and those will be unsed imports. Also, importing is private member of a library is never good as it may change in future updates.

**Comment by trusktr** [comment]
I've opened a request to fix issues like these in TypeScript by bringing declaration files to parity with language features. https://github.com/microsoft/TypeScript/issues/35822
